File: hackerrank/30 Days of Code/Day 2: Operators/solution.erl
-module(solution).
-export([main/0]).

                                                % Complete the solve function below.
solve(MealCost, TipPercent, TaxPercent) ->
    Tip = MealCost * (TipPercent / 100),
    Tax = MealCost * (TaxPercent / 100),
    TotalCost = MealCost + Tip + Tax,

    io:fwrite("~B~n", [round(TotalCost)]).

main() ->
    {MealCost, _} = string:to_float(string:chomp(io:get_line(""))),
    {TipPercent, _} = string:to_integer(string:chomp(io:get_line(""))),
    {TaxPercent, _} = string:to_integer(string:chomp(io:get_line(""))),

    solve(MealCost, TipPercent, TaxPercent),

    ok.

File: hackerrank/30 Days of Code/Day 2: Operators/solution.c
#include <stdio.h>
#include <math.h>

// Complete the solve function below.
void solve(double meal_cost, int tip_percent, int tax_percent) {
  double tip = meal_cost * (tip_percent / 100.0);
  double tax = meal_cost * (tax_percent / 100.0);
  int total_cost = (int) round(meal_cost + tip + tax);

  printf("%d\n", total_cost);
}

int main() {
  double meal_cost;
  int tip_percent;
  int tax_percent;

  scanf("%lf", &meal_cost);
  scanf("%d", &tip_percent);
  scanf("%d", &tax_percent);

  solve(meal_cost, tip_percent, tax_percent);
}
